test/avocado/run-tests

```diff
@@ -236,7 +238,7 @@ def main():
     parser.add_argument('-v', '--verbose', dest="verbosity", action='store_true',
                         help='Verbose output')
     parser.add_argument('-q', '--quick', action='store_true', help='Build faster')
-    parser.add_argument("-b", "--browser", choices=['none', 'firefox', 'chrome', 'explorer'],
+    parser.add_argument("-b", "--browser", choices=['none', 'firefox', 'chrome', 'explorer', 'MicrosoftEdge'],
```

Comment:
Can we please name this `edge` for consistency?

Reply:
Since this string gets passed straight to the selenium hub it'll need to be special cased then.

Reply:
I see. For our own mind's sake I think it wouldn't be too bad to translate `edge` into `MicrosoftEdge` in test/avocado/run-tests, WDYT?